temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
The provided pull request (PR) is creating a new Python file named `temp.py`. This file contains two lines of code:

1. An import statement for the `pdb` module, which is Python's debugger.
2. A print statement that outputs the string "hello".

Here's the content of the new file:
```python
import pdb
print("hello")
```

The user story description is missing, so the context of the business logic for which this change is made is not provided.

If you have any specific questions or need further assistance regarding this PR, feel free to ask!